### ISSUE TITLE
Colours found in settings, pilots seams to do nothing!

### ISSUE THREAD

**Issue by BoLtWD** [RotorHazard/RotorHazard#591]
In settings, pilots you have colours you can pick & assigned to each pilot, changing the colours does nothing regarding the colours assigned on the run paige. 

The colors on the run page seams fixed.

I don't know if this is a but or what the colour changes for pilots are associated with.

**Comment by HazardCreative** [comment]
Usage of pilot colors can be selected from the LED Effects panel.

**Comment by BoLtWD** [comment]
I take it from what you are saying is that the pilots colours are present as below, if that if the case under settings, pilots is the colours that you can select needed f so what for?

BLUE 
ORANGE 
GREEN
RED
YELLOW
DARK BLUE
LIGHT BLUE
GRAY

**Comment by HazardCreative** [comment]
Your last message is not clear. 

Colors are assigned using the method chosen for "Color Mode" in "LED Events" on the "Settings" page. The per-pilot colors are only used when this selector is set to "Pilot".

**Comment by BoLtWD** [comment]
Thank you,  I could not see anything in the documentation but that was a good chance it was me as I am good at missing things including the fpv gates LOL .

I don't think this is possible but as far as help documentation is there any way to add hyperlinks to send you or pop up documentation on the use of something.

Examples 

If I clicked on a hyperlink found in Frequency Setup tab under settings page this cold pop up the help for this Frequency Setup 
If I clicked on a hyperlink found in Race Management tab under run page this cold pop up the help for this Race Management.

The hyperlink to go to specific documentation could look like a question mark.

This would probably be a lot of work but just a thought.
